wix/Main.hotspot.wxs

```diff
@@ -35,30 +35,32 @@
 			<Custom Action="PreventDowngrading" After="FindRelatedProducts">NEWPRODUCTFOUND</Custom>
 		</InstallUISequence>
     -->
-    
+
     <!-- Define the directory structure -->
     <Directory Id="TARGETDIR" Name="SourceDir">
-      <?if $(env.Platform)=x86?>
-				<Directory Id="ProgramFilesFolder">
-          <Directory Id="PRODUCTMANUFACTURER" Name="$(var.ProductManufacturer)">
-            <Directory Id="INSTALLDIR" Name="$(var.AppFolder)" />
-          </Directory>
-				</Directory>
-			<?else?>
-				<Directory Id="ProgramFiles64Folder">
-          <Directory Id="PRODUCTMANUFACTURER" Name="$(var.ProductManufacturer)">
-            <Directory Id="INSTALLDIR" Name="$(var.AppFolder)" />
-          </Directory>
-				</Directory>
-			<?endif?>
+    <?if $(env.Platform)=x86?>
```

Comment:
again does this need to be `x86-32`?

Reply:
same as above

Reply:
if I set env.Platform to x86-32 the installer display x64 .. ( by default ) because he dont know wich platform it is

Reply:
and we proably do it better with sys.BUILDARCH